messaging/src/test/java/org/frankframework/extensions/kafka/KafkaIntegrationTest.java

```diff
@@ -0,0 +1,75 @@
+package org.frankframework.extensions.kafka;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.HashMap;
+import java.util.Objects;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.common.serialization.ByteArraySerializer;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.kafka.KafkaContainer;
+
+import org.frankframework.core.PipeLineSession;
+import org.frankframework.receivers.RawMessageWrapper;
+import org.frankframework.stream.Message;
+
+@Testcontainers(disabledWithoutDocker = true)
+@Tag("integration")
+public class KafkaIntegrationTest {
+
+	@Container
+	private static final KafkaContainer kafkaContainer = new KafkaContainer("apache/kafka");
+
+	final KafkaSender sender = new KafkaSender();
+
+	final KafkaListener listener = new KafkaListener();
+
+	@BeforeEach
+	void setUp() throws Exception {
+		sender.setTopic("test.test2");
+		sender.setClientId("test");
+		sender.setBootstrapServers(kafkaContainer.getBootstrapServers());
+		sender.configure();
+		sender.setProducer(new KafkaProducer<>(sender.properties, new StringSerializer(), new ByteArraySerializer()));
```

Comment:
Is het niet logischer om `sender.start()` te doen? Die zou toch de goede producer moeten aanmaken?